### PR TITLE
VACMS 4871: Switch from Yaml-Tasks to Task [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by ndouglas**
## Description

See #4871.  At present this is just a PoC to see what breaks and where, and if there are any deficits in Task relative to Yaml-Tasks that might make a different course of action necessary.

## Testing done
Ran some Lando commands manually, saw expected things.

## QA steps

As a developer:
1. Validate Acceptance Criteria from issue #4871:
    - [ ] Marks commit statuses as pending
    - [ ] Marks corresponding commit status as fail/pass when test suite is finished

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: scripts/install_task.sh [hunk 1]
```diff
@@ -0,0 +1,383 @@
+#!/bin/sh
```
Comment (acrollet) [inline scripts/install_task.sh:1]: minor comment, maybe this file could be renamed to `install_task_runner.sh` or similar to make its purpose more clear in e.g. the tugboat config?

Comment (ndouglas) [reply scripts/install_task.sh:1]: Sure.  I wish they'd named it anything _but_ "Task" 😖

Comment (acrollet) [reply scripts/install_task.sh:1]: even "tusk"

**Comment by ndouglas** [comment]
So according to [this branch](https://tugboat.vfs.va.gov/606374566a899402553e0667), things are building properly and not erroring, but the tests aren't running (at least not in any useful fashion).  So tomorrow I'm gonna work with the ReviewDog side of things and try to get reporting working properly.  It occurred to me that this no-base-preview thing might be a bit of a pain while working on something that directly affects the automated tests... so that's something else for me to figure out.

**Pull request closed without merging**